recipe/meta.yaml

```
@@ -1,52 +1,57 @@
 {% set name = "httpstan" %}
-{% set version = "4.6.1" %}
+{% set version = "4.13.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/stan-dev/httpstan/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 703e5e04e60651e0004574bb9695827d759fd13eb0d6bd67f827c1bfa0a1fd31
+  sha256: 6b15a07557715e79e6fd66993930003b270f8b8b0c9e65f84978afe5e6bb3047
 
 build:
-  number: 3
-  script: "{{ PYTHON }} -m pip install . -vv"
-  skip: True  # [py<37] 
-  
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+  skip: True  # [py<310]
+
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
     - {{ compiler('cxx') }}
   host:
     - pip
-    - poetry  >=1.1.4
-    - pybind11  >=2.6.2
+    - poetry-core >=1.0.0
+    - pybind11  >=2.9.2
     - python
+    - setuptools
   run:
-    - aiohttp  >=3.7
-    - appdirs  >=1.4
-    - lz4  >=3.1
-    - marshmallow  >=3.10
-    - numpy  >=1.16
     - python
-    - setuptools  >=41.0
-    - webargs  >=7.0
+    - aiohttp  >=3.7,<4.0
+    - appdirs  >=1.4,<2.0
+    - marshmallow  >=3.10,<4.0
+    - numpy  >=1.19
+    - webargs  >=8.0,<9.0
+    - setuptools >=48.0
 
 test:
   imports:
     - httpstan
     - httpstan.services
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
   home: https://pypi.org/project/httpstan/
   license: BSD-2-Clause
```

```suggestion
  license: ISC
```
https://github.com/stan-dev/httpstan/blob/4.13.0/LICENSE